index.bs

```diff
@@ -316,56 +317,90 @@ Interface {#interface}
 =================
 
 <pre class="idl">
-dictionary NavigatorUABrandVersionDict {
-  required DOMString brand;
+dictionary NavigatorUABrandVersion {
+  DOMString brand;
   DOMString version;
 };
 
+dictionary UADataOptions {
```

Comment:
Nit: I'm fine with this. I'd also be fine with passing an array of names (e.g. `['platform', 'platformVersion', 'architecture']`). The dictionary approach makes sense to me in a world where we expect one or more of these to be non-boolean (e.g. they need some options). If we expect to match the rest of client hints insofar as the name maps directly to a thing without options, the array makes more sense to me.

Reply:
OK. I originally went with an options object as it's presumably lower overhead, but this is not likely to be called multiple times, so it probably doesn't matter.